### PR TITLE
Feat/use new date picker in error metrics

### PR DESCRIPTION
**Pull request by betty-alagwu**
Hey @Vadman97 Thank you for letting me work on this.

This PR is a first pass at using the new datePicker component on the error metrics.

#5196

Here is a video of how it works.

https://user-images.githubusercontent.com/94234459/235759318-359073a8-b5bd-498d-9e08-988352759ca8.mp4

### CODE REVIEW

**Comment by betty-alagwu** [comment]
Currently investigating failing checks.

**Comment by et** [comment]
Hey @betty-alagwu - the failing checks are unrelated to your build. It was fixed in #5206. Merge main into this branch and your build should pass.

**Comment by betty-alagwu** [comment]
> Hey @betty-alagwu - the failing checks are unrelated to your build. It was fixed in #5206. Merge main into this branch and your build should pass.

Thank you @et. All checks are passing now.

**Comment by betty-alagwu** [comment]
Hello @Vadman97 
This PR is ready for review.

File: frontend/src/components/TimeRangePicker/TimeRangePicker.tsx [hunk 1]
```diff
@@ -1,304 +1,72 @@
...
-	{ offset: 360, label: 'Last 6 hours' },
-	{ offset: 1440, label: 'Last 24 hours' },
-	{ offset: 10080, label: 'Last 7 days' },
-	{ offset: 43200, label: 'Last 30 days' },
+import React, { useEffect, useState } from 'react'
+
+const presets = [
+	{
+		label: 'Last 5 minutes',
+		startDate: new Date(moment().subtract(5, 'minutes').format(FORMAT)),
```
Comment (Vadman97) [inline frontend/src/components/TimeRangePicker/TimeRangePicker.tsx:12]: great improvement!

**Review by Vadman97 — APPROVED** [review]
working great for me locally! thank you
image

File: frontend/src/components/TimeRangePicker/TimeRangePicker.tsx (hunk 1 above)
Comment (betty-alagwu) [reply frontend/src/components/TimeRangePicker/TimeRangePicker.tsx:12]: Thank you 🙏🏼

**Pull request merged**